src/app/tales/components/tale-run-button/tale-run-button.component.html

```diff
@@ -1,22 +1,32 @@
-<div>
-  <button *ngIf="!instance" class="right floated" [ngClass]="{ 'tale-button': !isPrimary, 'tale-button-primary': isPrimary }" (click)="startTale()">
-      <i class="play icon"></i>
-      Run Tale
-  </button>
-  <button *ngIf="instance && instance.status === 0" class="right floated" [ngClass]="{ 'tale-button': !isPrimary, 'tale-button-primary': isPrimary }" (click)="startTale()">
-      <i class="fas fa-spinner fa-pulse"></i>
-      Starting...
-  </button>
-  <button *ngIf="instance && instance.status === 1" class="right floated" [ngClass]="{ 'tale-button': !isPrimary, 'tale-button-primary': isPrimary }" (click)="stopTale()">
-      <i class="stop icon"></i>
-      Stop Tale
-  </button>
-  <button *ngIf="instance && instance.status === 2" class="right floated" [ngClass]="{ 'tale-button': !isPrimary, 'tale-button-primary': isPrimary }" (click)="stopTale()">
-      <i class="warning icon"></i>
-      Error
-  </button>
-  <button *ngIf="instance && instance.status === 3" class="right floated" [ngClass]="{ 'tale-button': !isPrimary, 'tale-button-primary': isPrimary }" (click)="stopTale()">
-      <i class="fas fa-spinner fa-pulse"></i>
-      Stopping...
-  </button>
-</div>
\ No newline at end of file
+<div>
+  <button *ngIf="tale && tale.status === 0" class="right floated" [ngClass]="{ 'tale-button disabled': !isPrimary, 'tale-button-primary disabled': isPrimary }" disabled="true">
+      <i class="fas fa-spinner fa-pulse"></i>
+      Importing...
+  </button>
+  <button *ngIf="tale && tale.status === 2" class="right floated" [ngClass]="{ 'tale-button disabled': !isPrimary, 'tale-button-primary disabled': isPrimary }" disabled="true">
+      <i class="warning icon"></i>
+      Import Failed
+  </button>

```

Comment:
```suggestion
  <button *ngIf="tale && tale.status === 0" class="right floated" [ngClass]="{ 'tale-button disabled': !isPrimary, 'tale-button-primary disabled': isPrimary }" disabled="true">
      <i class="fas fa-spinner fa-pulse"></i>
      Preparing...
  </button>
  <button *ngIf="tale && tale.status === 2" class="right floated" [ngClass]="{ 'tale-button disabled': !isPrimary, 'tale-button-primary disabled': isPrimary }" disabled="true">
      <i class="warning icon"></i>
      Initialization Failed
  </button>

```
Note that the "Error" label is already applied in some cases where the instance fails to launch. Such cases are correctable by deleting and relaunching (e.g. pressing the Error button and then clicking Run Tale again).

To distinguish such events from failed imports / failed copies, I'd suggest using something slightly more explicit here